spec/features/delete_scheduled_messages_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "Delete scheduled messages" do
+  scenario "from list of scheduled messages", :js do
```

Comment:
I think we only want admins (user with `admin` attribute set to true) to be able to delete messages. Let's write a test for that and implement it!